LEAF_Request_Portal/templates/view_search.tpl

```diff
@@ -1,5 +1,6 @@
 <section style="display: flex; flex-direction: column; width: fit-content;">
     <div id="searchContainer"></div>
+    <button id="btn_abortSearch" class="buttonNorm">Stop searching for more</button>
```

Comment:
Please check with Carrie to see if this needs a label to be 508 compliant.

Reply:
Added type button, it is a thing that may or may not be needed but helps inform tools that it is in fact a button even through it is a button element.